OctaneVSPlugin/Common/Collector/EntitiesCollector.cs

```diff
@@ -62,7 +62,15 @@ protected EntitiesCollector(OctaneServices service)
             // This allows us to later aggregate all the fetched entities without caring about the specific type of each one.
             collectorTask.ContinueWith(result =>
             {
-                tcs.TrySetResult(collectorTask.Result);
+                if (result.IsFaulted || result.IsCanceled)
+                {
+                    // return an empty list in case something went wrong with the collector task
+                    tcs.TrySetResult(new EntityListResult<TEntityType>());
```

Comment:
Is the fact that the task failed signaled to the user in some way?

Reply:
@atothhpe Right now the user won't know if there was an underlying issue. Will try to investigate a better way to report this to the user